quickstart-node/src/utils/url.utils.ts

```diff
@@ -0,0 +1,9 @@
+export const generateUrlSafeString = (length: number) => {
```

Comment:
Maybe change the name t o better understand the usage, maybe URL safe **random** string